volatility3/framework/plugins/windows/handles.py

```diff
@@ -18,13 +18,9 @@ class Handles(interfaces.plugins.PluginInterface):
     """Lists process open handles."""
 
     _required_framework_version = (2, 0, 0)
-    _version = (3, 0, 0)
+    _version = (4, 0, 0)
 
-    def __init__(self, *args, **kwargs):
-        super().__init__(*args, **kwargs)
-        self._type_map = None
-        self._cookie = None
-        self._level_mask = 7
+    LEVEL_MASK = 7
```

Comment:
Can't decide whether this is a constant that other plugins might want to access, or just an internal constant only for use by the plugin?  What do you think?

Reply:
I think it should probably be internal, I can't think of anything else that would have a use for it right now. Interestingly, it is defined as 3 instead of 7 in the reactOS source: https://github.com/reactos/reactos/blob/d722e7081796692dc2fb225517d2d9fb6283d64b/ntoskrnl/ex/handle.c#L64-L65

I tested against the regression set with both values and there were no output changes, so I guess it's fine for now regardless.